mountpoint-s3/tests/fuse_tests/statfs_test.rs

```diff
@@ -0,0 +1,58 @@
+use crate::common::fuse::{self, TestSessionCreator};
+use test_case::test_case;
+
+/// Tests that values we set are reported correctly.
+fn statfs_test_fuser_values(creator_fn: impl TestSessionCreator, prefix: &str) {
```

Comment:
The test name isn't testing fuser values, but rather we want to ensure that `statfs` is returning what we expect.

I'd remove `fuser` or replace with `static` or something. Just want to avoid misleading test name.

```suggestion
/// Tests that values we set are reported correctly.
fn statfs_test_static_values(creator_fn: impl TestSessionCreator, prefix: &str) {
```